test/python/visualization/timeline/test_events.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
```

Comment:
It seems like `load_program` isn't being tested?

Reply:
Right. We don't have #4555 merged and there is no input object for `load_program`. This will be added once it is merged.